pennylane/templates/subroutines/arithmetic/temporary_and.py

```diff
@@ -262,7 +262,21 @@ def _temporary_and(wires: WiresLike, **kwargs):
 
 
 add_decomps(TemporaryAND, _temporary_and)
-# TODO: add add_decomps("Adjoint(TemporaryAND)", _adjoint_TemporaryAND) when MCMs supported by the pipeline
+
+
+def _adjoint_temporary_and_resources(*, base_class=None, base_params=None, **__):
+    return {ops.Hadamard: 1, ops.MidMeasure: 1, ops.Conditional: 1}
```

Comment:
When you have a conditional operator, you would only declare the actual operator being conditionally applied (in this case, the `CZ`), not the `Conditional`.